src/types/dto/review.ts

```diff
@@ -29,7 +35,15 @@ export interface EditedMediaDto extends BaseEditedItemDto {
   type: "file" | "image"
 }
 
-export type EditedItemDto = EditedPageDto | EditedConfigDto | EditedMediaDto
+export interface EditedPlaceholderDto extends BaseEditedItemDto {
+  type: "placeholder"
+}
+
+export type EditedItemDto =
+  | EditedPageDto
+  | EditedConfigDto
+  | EditedMediaDto
+  | EditedPlaceholderDto
 
```

Comment:
is this meant to be here? i thought we shifted over to processing and not returning to FE? (the `filter` stuff in `reviewRequestService`)

Reply:
placeholder file changes should still show up on `compareDiff` and `compareDiffWithMappings`, creating new type of `DisplayedEditedItemDto` without `EditedPlaceholderDto` when returning to FE